frontend/src/Store/Actions/Settings/general.ts

```diff
@@ -3,6 +3,7 @@ import createFetchHandler from 'Store/Actions/Creators/createFetchHandler';
 import createSaveHandler from 'Store/Actions/Creators/createSaveHandler';
```

Comment:
The file extension change triggered different eslint formatting rules

Reply:
Eventually the plan is to nuke all the redux, so you can drop these changes, we've only been typing the state so we can verify types on things we're consuming.

Reply:
Does that revert include the file renaming ? that would bring the type used back to `any`

And if the revert is only about the eslint changes, the checkup script would stop passing

I wouldn't mind dropping redux, but that MR is already doing way more than it should

Reply:
I didn't realize that using `payloadCreator`  in `setGeneralSettingsValue` lead to the type being changed, which leads to the issue you're seeing. For now I think we go with not typing these actions at all. and just use a expect error to ignore typescript.